app/views/fsss_view.py

```diff
@@ -102,6 +102,8 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
       # Device phone number
       'Unique Employee ID': get_emp_info('uniqueEmployeeId'),
       # Device Type
```

Comment:
This line can be removed now - it was a dummy to represent the fact that I couldn't find a Device Type entry to put here.

Reply:
Deleted